src/res_parsetree_viewer.ml

```diff
@@ -10,11 +10,12 @@ let arrowType ct =
       let arg = ([], lbl, typ1) in
       process attrsBefore (arg :: acc) typ2
     | {
-     ptyp_desc = Ptyp_arrow ((Nolabel as lbl), typ1, typ2);
-     ptyp_attributes = [({txt = "bs" | "res.async"}, _)] as attrs;
+     ptyp_desc = Ptyp_arrow (Nolabel, _typ1, _typ2);
+     ptyp_attributes = [({txt = "bs" | "res.async"}, _)];
```

Comment:
Can you add examples of nested async functions?

Reply:
Not sure what that is doing here. `type f = async (…) => …` does not exist. Will remove.

Reply:
Done in https://github.com/rescript-lang/syntax/pull/668/commits/def9fa087ac93f0b9857e0e20c2de4fd6b96ce82